guides/plugins/plugins/administration/overriding-routes.md

```diff
@@ -0,0 +1,59 @@
+# Overriding routes
+
+## Overview
+
+In the `Administration` core code, each module is defined in a directory called `module`. Modules define routes wich can be extended with `routeMiddleware`.
+Learn what else you can customize, in existing modules in this [guide](./customizing-modules.md)
+
+A `module` is an encapsulated unit which implements a whole feature. For example there are modules for customers, orders, settings, etc.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance. Of course, you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/) and the [Vue Router](https://router.vuejs.org/). However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Appling the override
+
+At some point you might want to override or change existing routes, to for example, change the privileges required for a route or entirely replace it with your own.
+
+This is done by creating a new module and implementing a `routeMiddleware`.
```

Comment:
```suggestion
This is done by creating a new module and implementing a `routeMiddleware` method.
```
Can this bew called a  `method` ?

Reply:
it's a middleware method, so technically jes, but i would leave it as it is